internal/app/utils.go

```diff
@@ -482,6 +482,27 @@ func decryptSecret(name string) error {
 		if settings.EyamlPrivateKeyPath != "" && settings.EyamlPublicKeyPath != "" {
 			args = append(args, []string{"--pkcs7-private-key", settings.EyamlPrivateKeyPath, "--pkcs7-public-key", settings.EyamlPublicKeyPath}...)
 		}
+	} else if settings.VaultEnabled {
+		args = []string{"vault", "dec"}
```

Comment:
This will fail if the vault plugin is missing and the resulting error might not be easy to understand, we should check if the plugin ins installed during the DSF validation if `settings.VaultEnabled` is set. You can use the `helmPluginExists` function for this.

Reply:
Pushed some commits that should resolve these issues, thanks for the feedback.